src/components/accordion/index.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames'
+import AccordionDisclosure from 'components/accordion-disclosure'
+import type { AccordionProps, AccordionContentItem } from './types'
+import s from './accordion.module.css'
+
+/**
+ * Renders a group of AccordionDisclosure items.
+ */
+function Accordion({
+	className,
+	items,
+	activatorHeadingLevel,
+}: AccordionProps) {
+	const itemsCount = items.length
+	return (
+		<div className={classNames(s.root, className)}>
+			{items.map(({ title, content }: AccordionContentItem, idx: number) => {
+				return (
+					<AccordionDisclosure
+						key={title}
+						title={title}
+						groupData={{
+							numItems: itemsCount,
+							currentIndex: idx,
+						}}
+						activatorHeadingLevel={activatorHeadingLevel}
+					>
+						{content}
+					</AccordionDisclosure>
+				)
+			})}
+		</div>
```

Comment:
I think we need some tweaks to the semantics here so that an `Accordion` is conveyed as a group. I think there's also a lot of value in supporting `aria-label`/`aria-labelledby`, but that's a bit trickier when it comes to exposing this as an authoring component, so I think we can iterate on that part later.

An easy-win for achieving the group semantics, while also leaving room for `aria-label`/`aria-labelledby` later, feels like using `<ul>` and `<li>` elements. What do you think?

Reply:
I don't have a strong opinion on using `<ul>` & `<li>`! My understanding was that the use of headings would create the correct semantics to match [the Accordion pattern](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/).

For adding `<ul>` & `<li />`, would the implication here be to make `s.root` here a `<ul>` and then wrap each `AccordionDisclosure` in an `<li />`?